src/OrbitCore/Path.cpp

```diff
@@ -88,6 +92,18 @@ std::filesystem::path CreateOrGetOrbitAppDataDir() {
   return path;
 }
 
+std::filesystem::path CreateOrGetOrbitUserDataDir() {
+#ifdef WIN32
+  std::filesystem::path path = std::filesystem::path(GetEnvVar("USERPROFILE"));
+#else
+  std::filesystem::path path = std::filesystem::path(GetEnvVar("HOME"));
+#endif
+  path = path / "Documents" / "Orbit";
```

Comment:
what would be correct way to detect Documents folder? is there another environment variable for it?
(we may want to make it configurable, but this is probably for separate PR)

Reply:
AFAIK https://docs.microsoft.com/en-us/windows/win32/api/shlobj_core/nf-shlobj_core-shgetfolderpatha should be the correct Win32 API function. Alternatively you could also read it from the registry or use Qt's https://doc.qt.io/qt-5/qstandardpaths.html. The latter is probably easiest if you can inject the path and not use a free function on OrbitCore to determine it.

Reply:
I think I am going to leave it for latter PRs. It looks like this can be done easier if we moved from global functions to a Locations object that initialized and passed around and this requires some refactoring.